Source/Core/InputCommon/CMakeLists.txt

```diff
@@ -101,13 +101,21 @@ if(WIN32)
     ControllerInterface/DInput/XInputFilter.h
     ControllerInterface/Win32/Win32.cpp
     ControllerInterface/Win32/Win32.h
-    ControllerInterface/WGInput/WGInput.cpp
-    ControllerInterface/WGInput/WGInput.h
     ControllerInterface/XInput/XInput.cpp
     ControllerInterface/XInput/XInput.h
     ControllerInterface/ForceFeedback/ForceFeedbackDevice.cpp
     ControllerInterface/ForceFeedback/ForceFeedbackDevice.h
   )
+	if(NOT MSVC)
+  	target_compile_definitions(inputcommon PRIVATE WINVER=0x0602)
+	endif()
+  if(MSVC)
+    target_sources(inputcommon PRIVATE
```

Comment:
Can't it be simplified using `if() ... else() ... endif()` instead?

Reply:
done